adapter/stdioLogger/stdioLogger.go

```diff
@@ -17,19 +17,36 @@
 package stdioLogger
 
 import (
-	"encoding/json"
-	"io"
-	"os"
+	"fmt"
 
-	me "github.com/hashicorp/go-multierror"
+	multierror "github.com/hashicorp/go-multierror"
+	"go.uber.org/zap"
+	"go.uber.org/zap/zapcore"
 
 	"istio.io/mixer/adapter/stdioLogger/config"
 	"istio.io/mixer/pkg/adapter"
 )
 
 type (
 	builder struct{ adapter.DefaultBuilder }
-	logger  struct{ logStream io.Writer }
+	logger  struct {
+		outputPaths []string
+		impl        *zap.Logger
+	}
+	zapBuilderFn func(outputPaths ...string) (*zap.Logger, error)
+)
+
+const (
+	stdErr = "stderr"
+	stdOut = "stdout"
```

Comment:
Super-nit: I question whether these actually add value: we only use them once in stdioLogger, and only a few times in the tests. It's not like stdout/stderr are likely to change any time soon.